src/libtest/lib.rs

```diff
@@ -485,9 +485,7 @@ pub fn run_test(
         }
         StaticBenchFn(benchfn) => {
             // Benchmarks aren't expected to panic, so we run them all in-process.
-            crate::bench::benchmark(desc, monitor_ch, opts.nocapture, |harness| {
-                (benchfn.clone())(harness)
-            });
+            crate::bench::benchmark(desc, monitor_ch, opts.nocapture, |harness| (benchfn)(harness));
```

Comment:
You don't need the parens around `benchfn` - in fact, you might be able to just pass `benchfn` to `crate::bench::benchmark` directly, without the closure wrapping.

Reply:
Updated.